package.json

```diff
@@ -107,8 +107,8 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
-    "@tableland/evm": "^4.2.0",
-    "@tableland/sqlparser": "^1.0.5",
+    "@tableland/evm": "^4.2.2",
+    "@tableland/sqlparser": "^1.0.6-pre.3",
```

Comment:
added the new evm tableland dep with sepolia network, and a prerelease of the sqlparser